docs/tutorials/13_trotterQRTE.ipynb

```diff
@@ -0,0 +1,983 @@
+      "text/plain": [
+       "<Figure size 640x480 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "fig, axes = plt.subplots(3, sharex=True)\n",
+    "times = np.linspace(0, final_time, num_timesteps + 1) # includes initial state\n",
+    "axes[0].plot(times, observables[:, 0], label='First order', marker='x', c='darkmagenta', ls='-', lw=.8)\n",
+    "axes[1].plot(times, observables[:, 1], label='First order', marker='x', c='darkmagenta', ls='-', lw=.8)\n",
+    "axes[2].plot(times, observables[:, 2], label='First order', marker='x', c='darkmagenta', ls='-', lw=.8)\n",
+    "axes[0].set_ylabel(\"Energy\")\n",
+    "axes[1].set_ylabel(\"Magnetization\")\n",
+    "axes[2].set_ylabel(\"Mean spin correlation\")\n",
+    "axes[2].set_xlabel(\"Time\")\n",
+    "fig.suptitle(\"Observable evolution\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "063a380f-be6e-4664-849b-55dff23153b6",
+   "metadata": {},
+   "source": [
+    "Let us verify these results by comparing these expected values using Trotter evolution with the exact ones. To that end, we evaluate directly the expression discussed in the introduction,\n",
+    "\n",
+    "$$\n",
+    "\\vert \\psi(t) \\rangle = e^{-i H t} \\vert \\psi(0) \\rangle \\text{,}\n",
+    "$$\n",
+    "\n",
+    "on each one of the timesteps used by Trotter. We compute this exponential using scipy's [`linalg.expm`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.expm.html) function, and then we let the initial system evolve using the `Statevector`'s [`evolve` method](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Statevector.evolve.html). This becomes too costly to be performed on larger systems very quickly."
```

Comment:
```suggestion
    "on each one of the timesteps used by Trotter. We compute this exponential using SciPy's [linalg.expm](https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.expm.html) function, and then we let the initial system evolve using the `Statevector`'s [evolve](https://qiskit.org/documentation/stubs/qiskit.quantum_info.Statevector.evolve.html) method. This becomes too costly to be performed on larger systems very quickly."
```
![image](https://github.com/qiskit-community/qiskit-algorithms/assets/40241007/52fd221f-5682-4f63-b804-0b269dde72e7)

I also change the `scipy` to `SciPy` since the latter is how the project itself has it